draft-birkholz-scitt-architecture.md

```diff
@@ -162,56 +162,56 @@ Claim:
 
 Issuer:
 
-: creator of Claims submitted to a Transparency Service for Registration. The Issuer may be the owner or author of the Artifact, or a completely independent third party.
+: creator of Claims submitted to a Receipt Service for Registration. The Issuer may be the owner or author of the Artifact, or a completely independent third party.
 
 Envelope:
 
 : the metadata added to the Statement by the Issuer to make it a Claim. It contains the identity of the Issuer and other information to help Verifiers identify the Artifact referred in the Statement. A Claim binds the Envelope to the Statement. In COSE, the Envelope consists of protected headers.
 
-Feed:
+Ledger:
 
-: An identifier chosen by the Issuer for the Artifact. For every Issuer and Feed, the Ledger on a Transparency Service contains a sequence of Claims about the same Artifact.
- In COSE, Feed is one header attributes in the protected header of the Envelope.
+: an optional verifiable data structure that stores Claims in a Receipt Service. SCITT supports multiple Ledger formats to accommodate different Receipt Service implementations, such as historical Merkle Trees and sparse Merkle Trees. Not all Receipt Services require a Ledger.
 
-Ledger:
+Feed:
 
-: the verifiable data structure that stores Claims in a transparency service. SCITT supports multiple Ledger formats to accommodate different transparency service implementations, such as historical Merkle Trees and sparse Merkle Trees.
+: An identifier chosen by the Issuer for the Artifact. For every Issuer and Feed, the Ledger on a Receipt Service contains a sequence of Claims about the same Artifact.
+In COSE, Feed is one header attributes in the protected header of the Envelope.
 
-Transparency Service:
+Receipt Service (RS):
 
-: the entity that maintains and extends the Ledger, and endorses its state. A Transparency Service can be a complex distributed system, and SCITT requires the TS to provide many security guarantees about its Ledger. The identity of a TS is captured by a public key that must be known by Verifiers in order to validate Receipts.
+: the entity that maintains and extends the Ledger, and endorses its state. A Receipt Service can range from something as simple as a timestamping service to a more complex distributed system. The security properties of the RS depend on the type of verifiability it affords to verifiers. The identity of a RS is captured by a public key that must be known by Verifiers in order to validate Receipts.
```

Comment:
The security property is very underspecified. As written, the baseline guarantee is that the registration policy was satisfied when the claim was registered. Additional guarantees stemming from the addition of the claim to the ledger depend on the use of a ledger, the type of ledger/receipt, and how the server maintains and distributes the ledger.